src/microbe_stage/Patch.cs

```diff
@@ -32,15 +32,28 @@ public class Patch
     [JsonProperty]
     private Deque<PatchSnapshot> history = new Deque<PatchSnapshot>();
 
+#pragma warning disable 169 // Used through reflection
+    [JsonProperty]
+    private string untranslatedName;
+#pragma warning restore 169
+
+    public Patch()
+    {
+    }
+
     public Patch(string name, int id, Biome biomeTemplate)
     {
         Name = name;
         ID = id;
         BiomeTemplate = biomeTemplate;
         currentSnapshot.Biome = (BiomeConditions)biomeTemplate.Conditions.Clone();
+
+        TranslationHelper.CopyTranslateTemplatesToTranslateSource(this);
+        TranslationHelper.ApplyTranslations(this);
     }
 
-    [JsonProperty]
+    [JsonIgnore]
+    [TranslateFrom("untranslatedName")]
```

Comment:
Already mentioned, but if there isn't an easy way to fix the saves, you need to add a save breakage point for 0.5.3.2 in the `KnownSaveIncompatibilityPoints` list.

Reply:
I didn't find an issue about that, so I opened a new one: https://github.com/Revolutionary-Games/Thrive/issues/2152